### PR TITLE
Feature/mx 1475 integrate wikidata into seq repo

### PR DESCRIPTION
**Pull request by mr-kamran-ali**
# Added
- wikidata aux extractor into seq-repo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +49,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - receive one or None organization from wikidata aux extractor
 - adjust Timestamp usage to TemporalEntity
 - move quotation marks (") filtering to mex-common from requested wikidata label
+
+### Deprecated
+
```
Comment (erichesse) [inline CHANGELOG.md:54]:
```suggestion
```
don't need the category if it is empty

File: CHANGELOG.md [hunk 2]
```diff
@@ -60,6 +64,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - fix some docstring indents and typings
 - ifsg extractor
 
+### Security
+
```
Comment (erichesse) [inline CHANGELOG.md:68]:
```suggestion
```
same here

File: mex/seq_repo/transform.py [hunk 3]
```diff
@@ -79,6 +80,7 @@ def transform_seq_repo_distribution_to_extracted_distribution(
         seq_repo_sources: Seq Repo extracted sources
         seq_repo_distribution: Seq Repo extracted distribution
         mex_access_platform: Extracted access platform
+        extracted_organization_rki: wikdiata extractded organization
```
Comment (erichesse) [inline mex/seq_repo/transform.py:83]:
```suggestion
        extracted_organization_rki: wikidata extracted organization
```

File: mex/seq_repo/transform.py [hunk 4]
```diff
@@ -129,6 +130,7 @@ def transform_seq_repo_resource_to_extracted_resource(
         unit_stable_target_ids_by_synonym: Unit stable target ids by synonym
         project_coordinators_merged_ids_by_query_string: Seq Repo Sources resolved
                                                         project coordinators merged ids
+        extracted_organization_rki: wikdiata extractded organization
```
Comment (erichesse) [inline mex/seq_repo/transform.py:133]:
```suggestion
        extracted_organization_rki: wikidata extracted organization
```

**Review by erichesse — COMMENTED** [review]
Looks good! Just some cosmetic changes left

File: CHANGELOG.md (hunk 1 above)
Comment (mr-kamran-ali) [reply CHANGELOG.md:54]: But this one has a line 55 under this category: - get seq-repo data via mex-drop connector (was: file)

File: CHANGELOG.md (hunk 2 above)
Comment (mr-kamran-ali) [reply CHANGELOG.md]: fixed. 752f8a3f11243fb2f9cb7e82a581db8ebf5e8a48

File: mex/seq_repo/transform.py (hunk 4 above)
Comment (mr-kamran-ali) [reply mex/seq_repo/transform.py]: updated: f7c5a68101162b25c2e4aba74b7cafa9bc084cd9

File: mex/seq_repo/transform.py (hunk 3 above)
Comment (mr-kamran-ali) [reply mex/seq_repo/transform.py]: updated: f7c5a68101162b25c2e4aba74b7cafa9bc084cd9

**Pull request merged**